.github/workflows/coderabbit-integration.yml

```
@@ -0,0 +1,304 @@
+    outputs:
+      should-review: ${{ steps.check.outputs.should-review }}
+      
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          
+      - name: Check if review is needed
+        id: check
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:6:33: Double quote to prevent globbing and word splitting

(shellcheck)